### PR TITLE
Added BTN private tracker support.

### PR DESCRIPTION
**Pull request by smithjw**
Added BTN support to the latest version of SickBeard thanks to [tubbo](https://github.com/tubbo). I just went through his original fork [here](https://github.com/tubbo/Sick-Beard), and brought his changes up to date with the latest master version. All works here and have been using it for a couple of months with no issues.

### CODE REVIEW

**Comment by smithjw** [comment]
How long does it usually take for you to pull a request in?

**Comment by aaronflorey** [comment]
Forever, if at all.

**Comment by smithjw** [comment]
Well I can hope can't I. :)

**Comment by aaronflorey** [comment]
lol yeah. I forked your repo and ill keep it upto date so others can use it at btn

**Comment by midgetspy** [comment]
What kind of site is this - how much TV is on it? How many users?

**Comment by aaronflorey** [comment]
it's a torrent site for tv shows with 29k users and 56k torrents

http://omg.btnf.tw/r3dM

**Comment by smithjw** [comment]
I love our little community. Do you think there is a way to enable Sick Beard to search the back catalog of BTN or will it always be stuck with just the newly added content?

**Comment by aaronflorey** [comment]
For now it's just new content, but i may work on an api to allow backlog capabilities.

**Comment by smithjw** [comment]
Sweet, that would be amazing.

File: sickbeard/providers/btn.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        tvcache.TVCache.__init__(self, provider)
+
+        # only poll BTN every 15 minutes max
+        self.minTime = 15
+		
+
+    def _getRSSData(self):
+        # These will be ignored on the serverside.
+        ignore_regex = "all.month|month.of|season[\s\d]*complete"
```
Comment (midgetspy) [inline sickbeard/providers/btn.py:57]: This isn't used anywhere, is that intentional?

**Comment by smithjw** [comment]
I'm not sure to be honest, as I said in the description, I've just taken tubbo's changes and put them into the latest version so that I could use it with Qouch. :).

**Pull request merged**